data/css/modules/window/_simple.scss

```diff
@@ -4,11 +4,12 @@
 
     scrollbar {
         padding: 3px;
-        background-color: transparentize(black, 1 - 0.20);
+        background-color: #c3c4c4;
```

Comment:
I tried this out on history.en and animals.en as representative preset A and B apps. In preset B it looked out of place in my opinion. The background color of the scrollbar was darker, and that makes the border stand out too much. However, I can't find where the background color was overridden (not in preset_b.scss, nor animals.en's overrides.scss) so maybe I'm testing wrong and there's nothing to do here?

In preset A it would normally look fine, I think, except when there's a darker-colored background such as in history.en; in that case the scrollbar stands out because it's pretty light-colored. Consider moving the previous color into some of the app overrides for preset A apps with darker-colored backgrounds?